webapp/stores/emoji_store.jsx

```diff
@@ -148,8 +149,7 @@ class EmojiStore extends EventEmitter {
 
     getEmojiImageUrl(emoji) {
         if (emoji.id) {
-            // must match Client.getCustomEmojiImageUrl
-            return `/api/v3/emoji/${emoji.id}`;
+            return Client.getCustomEmojiImageUrl(emoji.id);
```

Comment:
273ccd93f2f978784ad1f35442217fe1f16bf767: Added an alias to substitute `client/web_client.jsx` with `client/test_web_client.jsx`, which works on Node.js but has limited functionality, for testing.